### ISSUE TITLE
 [Settings] | Website Section - Date & Time, DateTimeForm option labels not displaying formatted date and time

### ISSUE THREAD

**Issue by blesildaramirez** [pkp/pkp-lib#11079]
### Description
The `DateTimeForm` component needs to be updated to align with the new date-time format introduced in https://github.com/pkp/pkp-lib/issues/9733 & https://github.com/pkp/pkp-lib/issues/10264. There should be sample dates as option labels when displaying the form. It should use the new datetime conversion from `ui-library/src/utils/convertPhpDateTimeFormatToLuxon.js`.

Also, remove `moment` as an npm dependency as it's been replaced by `luxon` on #9733 .
The "special characters" link should also point to https://www.php.net/manual/en/datetime.format.php#refsect1-datetime.format-parameters and not the [old link](https://www.php.net/manual/en/function.strftime.php#refsect1-function.strftime-parameters). 

### Steps to Reproduce
In Settings->Website->Setup->Date & Time section, the field labels should display an example date&time based on the current date time from the page has loaded. Currently, the form displays the actual format names, not the formatted date and time:

![Image](https://github.com/user-attachments/assets/8eab1b95-efd6-4ab6-a131-e25d29270af5)

### Expected Result
Make the option labels use the formatted date and time, instead of the datetime format values, like in 3.4:

![Image](https://github.com/user-attachments/assets/3f569580-c535-431a-8683-5fffb3940610)

### Application Version
 OJS, OMP or OPS version 3.5.0

---
Pull requests:

**_stable-3_5_0_**
ui-library: https://github.com/pkp/ui-library/pull/557
pkp-lib: https://github.com/pkp/pkp-lib/pull/11081
ojs: https://github.com/pkp/ojs/pull/4740
omp: https://github.com/pkp/omp/pull/1896
ops: https://github.com/pkp/ops/pull/913
_Previous successful runs:_ ✅ 
ojs: https://github.com/pkp/ojs/actions/runs/13827983999 
omp: https://github.com/pkp/omp/actions/runs/13828074069
ops: https://github.com/pkp/ops/actions/runs/13828129356

**_main_**
ui-library: https://github.com/pkp/ui-library/pull/558
pkp-lib: https://github.com/pkp/pkp-lib/pull/11082
ojs: https://github.com/pkp/ojs/pull/4741 
omp: https://github.com/pkp/omp/pull/1897
ops: https://github.com/pkp/ops/pull/914
_Previous successful runs:_ ✅ 
ojs: https://github.com/pkp/ojs/actions/runs/13828386055
omp: https://github.com/pkp/omp/actions/runs/13828456808
ops: https://github.com/pkp/ops/actions/runs/13828513446

**Comment by blesildaramirez** [comment]
Hi @Tribunal33 , this is now ready for testing alongside https://github.com/pkp/pkp-lib/issues/9733. Let me know if you have any questions.

**Comment by Tribunal33** [comment]
This passes QA and tests have been added to regression suite.  Properly parsed examples are seen instead of format symbols.  Full test summary is part of parent ticket #9733
